bin/cron/ops_data_pull

```diff
@@ -54,7 +54,7 @@ SeamlessDatabasePool.use_persistent_read_connection do
         row.merge!(level_hash)
         row.merge!({:"Stage #{stage.position} Percent Complete" => percent[stage.position - 1].to_s})
       end
-      row.merge!({:'Script Percent Complete' => percent_complete_total(script, user).to_s})
+      row[:'Script Percent Complete'] = percent_complete_total(script, user).to_s
```

Comment:
Unclear from this context, since symbols are treated differently
equality-wise from strings.

```
Brians-MacBook-Pro:apps brian$ irb
@irb(main):001:0> :a == 'a'
=> false
```

Would need to chase down the key consumer's context, entirely possible it
is required to be a symbol. I'd imagine since this script went to lengths
to make a symbol-with-spaces it might be required


On Fri, Feb 26, 2016 at 10:17 AM, ashercodeorg <notifications@github.com>
wrote:

> In bin/cron/ops_data_pull
> <https://github.com/code-dot-org/code-dot-org/pull/7006#discussion_r54280038>
> :
>
> > @@ -54,7 +54,7 @@ SeamlessDatabasePool.use_persistent_read_connection do
> >          row.merge!(level_hash)
> >          row.merge!({:"Stage #{stage.position} Percent Complete" => percent[stage.position - 1].to_s})
> >        end
> > -      row.merge!({:'Script Percent Complete' => percent_complete_total(script, user).to_s})
> > +      row[:'Script Percent Complete'] = percent_complete_total(script, user).to_s
>
> Are the colon and quotes both necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/7006/files#r54280038>.
>